gulpfile.js/tasks/build.js

```diff
@@ -28,7 +27,7 @@ const buildSass = () => {
     )
     .pipe(rename({ basename: 'standalone' }))
     .pipe(dest(outDir))
-    .pipe(postcss([cssnano()]))
+    .pipe(postcss(postcssConfig.plugins))
```

Comment:
Is this change necessary for this PR?

Reply:
Reverted. If desired, I can add a card to move `cssnano` into the postcss config, and have it conditional to run only on build. I'm not clear whether this overrides the automatic import of other plugins, the way we currently have it. I believe that's what @calebeby was thinking.